corporate/views/installation_activity.py

```diff
@@ -184,9 +184,6 @@ def realm_summary_table(export: bool) -> str:
                     AND is_active=True
                     AND role IN %(admin_roles)s
             ) as realm_admin_user ON realm.id = realm_admin_user.realm_id
-        WHERE
-            _14day_active_humans IS NOT NULL
-            or realm.plan_type = 3
```

Comment:
Is removing this intended? seems like it might have been for testing.

Reply:
Oops, updated to add that back.